jobloop/cron.go

```diff
@@ -65,6 +65,18 @@ func (i cronJobImpl) ProcessOne() error {
 	return err
 }
 
+// ProcessOne implements the Job interface.
+func (i cronJobImpl) Process(count int) error {
+	for k := 1; k < count; k++ {
```

Comment:
This runs once fewer than you would expect. You want either `k := 0` or `k <= count`.